code/API_definitions/Traffic_Influence.yaml

```diff
@@ -79,18 +82,24 @@ info:
     by a source port and a destination port and protocol. To optimise the flow
     from more source ports or destination ports or protocols, the TI API must be
     invoked many times.\
+    It is important to notice that it is a task of the Service Application to
```

Comment:
The "Service Application" seems to be a new name for existing concept like User Client or Client Application. If this is correct please check if we can reuse the existing name else we will need to define the term "Service Application".

Reply:
You are totally right, I will update it with "Client Application".